Dockerfile

```diff
@@ -1,9 +1,9 @@
 # fetch basic image
-FROM us.icr.io/wh-common-rns/base-images/ubi8-ibmjre:20220124.2042
+FROM us.icr.io/wh-common-rns/base-images/ubi8-ibmjre:20220317.0842
 
 WORKDIR /
 
-COPY target/cql-translation-server-1.5.1-2-jar-with-dependencies.jar cql-translation-server-jar-with-dependencies.jar
+COPY target/cql-translation-server-1.5.6-SNAPSHOT-jar-with-dependencies.jar cql-translation-server-jar-with-dependencies.jar
```

Comment:
Wonder if you could try "COPY target/*-jar-with-dependencies.jar ..." so you don't have to update it when the version changes or a release is done.

Reply:
Good suggestion - simple way to avoid dual maintenance when updating to new versions in the future.